docs/web-chat/internationalization.mdx

```diff
@@ -100,15 +100,15 @@ window.helpshiftConfig = {
 };
 ```
 
-Alternatively, you can call the `setLanguage` API to set the language after the initialization code.
+Alternatively, you can call the `setLanguage` API (Deprecated) to set the language after the initialization code.
 
 ```javascript
 Helpshift("setLanguage", "fr");
 ```
 
-<Admonition type="info" title="Note">
+<Admonition type="danger" title="Note">
 
-The preferred way to set the language is to use the `helpshiftConfig` object over the `setLanguage` API.
+The preferred way to set the language is to use the `helpshiftConfig` object. `setLanguage` API is deprecated.
```

Comment:
The preferred way to set the language is to use the `helpshiftConfig` object because the`setLanguage` API is deprecated.